transforms/acceptance/tracking_page_event_migration.js

```diff
@@ -0,0 +1,66 @@
+const {
+  addImport
+} = require('../../utils/imports');
+
+function substituteEvent(j, code, eventName) {
+  let needsImport = false;
+  code = j(code).find(j.CallExpression, {
+    callee: {
+      type: 'MemberExpression',
+      object: { type: 'ThisExpression' },
+      property: { type: 'Identifier', name: eventName }
+    }
+  })
+  .forEach(path => {
+    needsImport = true;
+    j(path).replaceWith(
+        j.callExpression(
+          j.identifier(path.node.callee.property.name),
+          path.node.arguments
+        )
+    );
+  }).closest(j.CallExpression, {
+    callee: { name: 'module' }
+  }).forEach(path => {
+    let setupApplicationTestExists =  j(path).find(j.Identifier, { name: 'setupApplicationTest' }).length !== 0;
+    let setupExists = j(path).find(j.Identifier, { name: 'setupIntercomEventService' }).length !== 0;
+    if (!setupExists) {
+      if (!setupApplicationTestExists) {
+        path.value.arguments[1].body.body.unshift("setupIntercomEventService(hooks);");
+        path.value.arguments[1].body.body.unshift("setupApplicationTest(hooks);");
```

Comment:
I don't think we need to add `setupApplicationTest` here. This is for the case where the helper is being setup in a submodule, in which case `setupApplicationTest` should already be done

Reply:
Ah got it ok.